### PR TITLE
doc: var name

### PR DESCRIPTION
**Pull request by DhariniJeeva**
This PR fixes the variable name that was changed recently

### CODE REVIEW

**Review by lisbet-alvarez — APPROVED** [review]

File: ui/admin/tests/e2e/README.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ You need to provide an SSH Key pair for the EC2 instance. We recommend creating
 
 [How to create EC2 SSH key pair](https://docs.aws.amazon.com/ground-station/latest/ug/create-ec2-ssh-key-pair.html).
 
+[How to create EC2 keypair using Doormat](https://eng-handbook.hashicorp.services/internal-tools/enos/common-setup-steps/#generate-an-aws-key-pair)
```
Comment (DhariniJeeva) [inline ui/admin/tests/e2e/README.md:85]:
```suggestion
[How to create EC2 key pair using Doormat](https://eng-handbook.hashicorp.services/internal-tools/enos/common-setup-steps/#generate-an-aws-key-pair)
```

**Review by lisbet-alvarez — APPROVED** [review]

File: ui/admin/tests/e2e/README.md [hunk 2]
```diff
@@ -82,6 +82,8 @@ You need to provide an SSH Key pair for the EC2 instance. We recommend creating
 
 [How to create EC2 SSH key pair](https://docs.aws.amazon.com/ground-station/latest/ug/create-ec2-ssh-key-pair.html).
 
+[How to create EC2 key pair using Doormat](https://eng-handbook.hashicorp.services/internal-tools/enos/common-setup-steps/#generate-an-aws-key-pair)
+
```
[the remark this answers is not in the document]
Comment (calcaide) [reply ui/admin/tests/e2e/README.md:86]: Nice!! I had no idea this resource was available, We can delete the `How to create EC2 SSH Key pair` above, I think is always better to rely on internal docs for this kind of things.

File: ui/admin/tests/e2e/README.md [hunk 3]
```diff
@@ -98,7 +100,8 @@ Enos needs some configuration variables to run the scenario successfully. [See t
 - `enos_user`: The user name to use for Boundary.
 - `tfc_api_token`: you need to provide the previously created token in Terraform cloud, there is no shared token within Boundary team.
 - `local_boundary_dir`: The directory that contains Boundary binary.
-- `local_boundary_ui_dir`: The directory that contains the copy of boundary-ui you want to use for UI tests.
+- `local_boundary_src_dir`: The directory that contains the copy of boundary you want to use for e2e tests
```
Comment (ZedLi) [inline ui/admin/tests/e2e/README.md:103]: Is this needed? I thought this was for testing the boundary BE e2e tests

**Review by ZedLi — APPROVED** [review]

File: ui/admin/tests/e2e/README.md (hunk 3 above)
Comment (DhariniJeeva) [reply ui/admin/tests/e2e/README.md:103]: I got an error when I didn't add these during the setup mentioned in [step](https://github.com/hashicorp/boundary-ui/tree/main/ui/admin/tests/e2e#setup-enos) in the [config](https://github.com/hashicorp/boundary/blob/main/enos/enos.vars.hcl)

Comment (DhariniJeeva) [reply ui/admin/tests/e2e/README.md:103]: Did it work for you without these changes? 🤔

**Review by cameronperera — APPROVED** [review]

File: ui/admin/tests/e2e/README.md (hunk 3 above)
Comment (ZedLi) [reply ui/admin/tests/e2e/README.md:103]: I only needed the UI parameter, though I only tried running our UI tests

Comment (calcaide) [reply ui/admin/tests/e2e/README.md:104]: `local_boundary_src_dir` is not need for UI e2e testing purposes. I tested and works no problem without `local_boundary_src_dir`

Comment (DhariniJeeva) [reply ui/admin/tests/e2e/README.md:104]: thanks for confirming, I will remove the field

**Pull request merged**